### PR TITLE
chore: push updated yarn.lock too

### PR DESCRIPTION
**Pull request by vladfrangu**

### CODE REVIEW

File: .github/workflows/docs.yaml [hunk 1]
```diff
@@ -59,7 +59,7 @@ jobs:
               uses: stefanzweifel/git-auto-commit-action@v5
               with:
                   commit_message: "chore: Automatic theme updating workflow [skip ci]"
-                  file_pattern: "website/package*.json"
+                  file_pattern: "website/package*.json website/yarn.lock"
```
Comment (B4nan) [inline .github/workflows/docs.yaml:62]: i guess we should do this in all the other repositories like sdks and clients

Comment (B4nan) [reply .github/workflows/docs.yaml:62]: how is this line about npm or yarn? all of them use some lockfile

Comment (vladfrangu) [reply .github/workflows/docs.yaml:62]: Yep, and the first glob matches everything for npm due to the *, we just needed to add the yarn lockfile here due to us using yarn

Comment (B4nan) [reply .github/workflows/docs.yaml:62]: oh right, so it was only an issue here, got it

**Review by B4nan — APPROVED** [review]

**Pull request merged**